codegen/service/testdata/views_code.go

```diff
@@ -0,0 +1,239 @@
+package testdata
+
+const ResultWithMultipleViewsCode = `// ResultTypeView is a type which is projected based on a view.
+type ResultTypeView struct {
+	A *string
+	B *string
+}
+
+// ResultType is the viewed result type that projects ResultTypeView based on a
+// view.
+type ResultType struct {
+	*ResultTypeView
+	// View to render
+	View string
```

Comment:
I wonder if we should not use a named field to hold `ResultTypeView`. if that type has ` View` field then this won't work.

Reply:
Now the projected type is held by the field named "Projected".